src/styles/main.scss

```diff
@@ -0,0 +1,62 @@
+/* #region Normalize */
+body {
+  margin: 0;
+}
+/* #endregion */
+
+/*#region Variables */
+$pin-size: 24px;
+/* #endregion */
+
+/* #region Mixins */
+
+@mixin position {
+  position: absolute;
+  left: 50%;
+  bottom: 50%;
+}
+
+@mixin pin_size($pin_size) {
+  width: $pin_size;
+  height: $pin_size;
+  border-radius: 50%;
+}
+
+/* #endregion */
+
+.minute {
+  @include position;
+  height: 175px;
+  width: 12px;
+  border-radius: 5px;
+  background: linear-gradient(rgb(150, 150, 150),#000);
+  animation: rotate 3600s linear infinite 2s;
+  transform-origin: bottom center;
+}
+
+.second {
+  @include position;
+  height: 300px;
+  width: 12px;
+  border-radius: 5px;
+  background: linear-gradient(rgb(150, 150, 150),#000);
+  animation: rotate 60s linear infinite 2s;
+  transform-origin: bottom center;
+}
+
+.pin {
+  @include position;
+  @include pin_size($pin-size);
+  background-color: black;
+  transform: translate3D(-25%, 50%, 0);
```

Comment:
you are not utilizing the Z axis. simple translate would do

Reply:
Fixed